### ISSUE TITLE
Wrong Task-list generation.

### ISSUE THREAD

**Issue by hazarek** [sbarex/QLMarkdown#23]
The task list looks like an ordinary list.
<img width="120" alt="Ekran Resmi 2020-12-29 22 19 50" src="https://user-images.githubusercontent.com/36481442/103308728-2b42ac80-4a24-11eb-8675-48b035a23083.png">

`ul.contains-task-list` class not defined in output html file. It cannot be manipulated with css. other lists will be affected. 

<img width="711" alt="Ekran Resmi 2020-12-29 21 40 15" src="https://user-images.githubusercontent.com/36481442/103307981-6c39c180-4a22-11eb-9946-55125fd9fef6.png">

markdown.css

```css
.markdown-body ul.contains-task-list {
  list-style: none;
  padding-left: 0;
}
```

**Comment by sbarex** [comment]
Oh, this is a bug of the cmark extension. I didn't develop tasklist myself. This confirms to me that github doesn't actually use his cmark-gfm to manage .md files.

I can try to fix but non in an elegant way (the cmark code it's quite obscure, at least for me who am not an expert in c programming).

**Comment by sbarex** [comment]
Fixed on 1.0b8.
